### PR TITLE
set proposer votes to 0 if no match returned from subgraph

### PR DESCRIPTION
**Pull request by ripe0x**
fixes miscounted proposer votes in candidate cards

### CODE REVIEW

File: packages/nouns-webapp/src/wrappers/nounsData.ts [hunk 1]
```diff
@@ -84,9 +84,10 @@ export const useCandidateProposals = () => {
   const threshold = useProposalThreshold() || 0;
   const candidatesData = proposerDelegates.data && unmatchedCandidates?.map(
     (candidate: PartialProposalCandidate, i: number) => {
-      const proposerVotes =
-        (proposerDelegates.data && proposerDelegates.data.delegates[i]?.nounsRepresented?.length) ||
-        0;
+      let proposerVotes = 0;
+      if (proposerDelegates.data && proposerDelegates.data.delegates.some(d => d.id === candidate.proposer.toLowerCase())) {
+        proposerVotes = proposerDelegates.data.delegates.find(d => d.id === candidate.proposer.toLowerCase())?.nounsRepresented?.length || 0;
+      }
```
Comment (solimander) [inline packages/nouns-webapp/src/wrappers/nounsData.ts:90]: nit: can just do `const proposerVotes = proposerDelegates.data?.delegates.find(d => d.id === candidate.proposer.toLowerCase())?.nounsRepresented?.length || 0;`

File: packages/nouns-webapp/src/wrappers/nounsData.ts [hunk 2]
```diff
@@ -84,9 +84,10 @@ export const useCandidateProposals = () => {
   const threshold = useProposalThreshold() || 0;
   const candidatesData = proposerDelegates.data && unmatchedCandidates?.map(
     (candidate: PartialProposalCandidate, i: number) => {
```
Comment (solimander) [inline packages/nouns-webapp/src/wrappers/nounsData.ts:86]: Remove unused `i`

File: packages/nouns-webapp/src/wrappers/nounsData.ts (hunk 1 above)
Comment (ripe0x) [reply packages/nouns-webapp/src/wrappers/nounsData.ts:90]: nice. ty!

**Comment by ripe0x** [comment]
thanks soli! made those changes and added a fix for the candidate proposer issue flagged by gami.

**Review by solimander — APPROVED** [review]

**Pull request merged**